verticapy/sql/create.py

```diff
@@ -98,6 +105,43 @@ def create_table(
     bool
         True   if  the  table  was   successfully
         created, False otherwise.
+
+    Examples
+    --------
+    .. ipython:: python
+
+        from verticapy.sql import create_table
+
+        # Generates the SQL needed to create the Table
+        create_table(table_name = "employees",
+                     schema = "public",
+                     dtype = {"name": "VARCHAR(60)", "salary": "FLOAT"},
+                     genSQL = True)
+
+        # Creates the table
+        create_table(table_name = "employees",
+                     schema = "public",
+                     dtype = {"name": "VARCHAR(60)", "salary": "FLOAT"})
+
```

Comment:
@tconsigny  you are creating the table twice.

Reply:
I thought when genSQL is set to true the SQL is generated but not executed?